sw.js

```diff
@@ -0,0 +1,38 @@
+---
+layout: null
+---
+
+importScripts( '{{ site.baseurl }}/cache-polyfill.js' );
+
+var filesToCache = [
+  // root
+  '{{ site.baseurl }}/',
+  '{{ site.baseurl }}/index.html',
+  // css
+  '{{ site.baseurl }}/assets/css/main.css',
+  '{{ site.baseurl }}/assets/css/normalize.css',
+  '{{ site.baseurl }}/assets/css/syntax.css',
+  // images
+  '{{ site.baseurl }}/assets/img/octocat.png',
+  // pages
+  '{{ site.baseurl }}/example_page/',
```

Comment:
It'd be nice if we can auto-populate the pages here so that the user doesn't have to manually update this file.

Try `{% for page in site.documents %}'{{ site.baseurl }}{{ page.url }}',{% endfor %}`.